<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Colors</class>
 <widget class="QDialog" name="Colors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVAMP: Colours</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="toolTip">
      <string>Choose the colour for the wildtype variants</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_1">
     <property name="toolTip">
      <string>Choose the colour for the wildtype variants</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>Choose the colour for the hetrozygous variants</string>
     </property>
     <property name="text">
      <string>01</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_2">
     <property name="toolTip">
      <string>Choose the colour for the hetrozygous variants</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>Choose the colour for the hetrozygous variants</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox_3">
     <property name="toolTip">
      <string>Choose the colour for the hetrozygous variants</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Choose the colour for the homozygous variants</string>
     </property>
     <property name="text">
      <string>11</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_4">
     <property name="toolTip">
      <string>Choose the colour for the homozygous variants</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>Choose the colour for the insertions and deletions</string>
     </property>
     <property name="text">
      <string>Indel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox_5">
     <property name="toolTip">
      <string>Choose the colour for the insertions and deletions</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>Choose the colour for the missing data</string>
     </property>
     <property name="text">
      <string>Missing</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBox_6">
     <property name="toolTip">
      <string>Choose the colour for the missing data</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
